### DISCUSSION TITLE
How can i make MAUI Blazor Application rounded and WindowStyle None

### DISCUSSION THREAD

**Discussion by wthks** [dotnet/maui#8045] [Q&A]
I m trying to figure out how i can make the maui blazor application rounded and without window style.
I could not figure out how that works. is there currently no possibility?

**Comment by wthks** [comment]
any info about this? I figured out that MAUI Blazor App uses `MauiWinUIApplication` does that not have support to change the Titlebar?

**Comment by mhrastegari** [comment]
You can change the colors of the titlebar like this in `Platforms/Windows/App.xaml` file.

```
<maui:MauiWinUIApplication
    x:Class="TodoTemplate.Client.App.Platforms.Windows.App"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:maui="using:Microsoft.Maui">
    <maui:MauiWinUIApplication.Resources>
        <ResourceDictionary>
            <SolidColorBrush x:Key="WindowCaptionForeground">#0D2960</SolidColorBrush>
            <SolidColorBrush x:Key="WindowCaptionBackground">#F2F2F2</SolidColorBrush>
            <SolidColorBrush x:Key="WindowCaptionForegroundDisabled">#0D2960</SolidColorBrush>
            <SolidColorBrush x:Key="WindowCaptionBackgroundDisabled">#F2F2F2</SolidColorBrush>
        </ResourceDictionary>
    </maui:MauiWinUIApplication.Resources>
</maui:MauiWinUIApplication>
```

**Reply by noobsoftware** [reply]
Is it possible to do something similar for MacCatalyst platform? Changing some of the properties of the window in AppDelegate seem to have no effect.

**Comment by NJullienSweet** [comment]
Still no answer in 2023?
I was given the task to make my company's app a custom style, by replacing the default Windows title bar (and its Minimize/Maximize/Close buttons) and add a custom radius to match our design guidelines.
I can't find any helping topic about this.
